conf/messages

```diff
@@ -1482,24 +1699,50 @@ acquisitionFees.trust.title=Were there any additional acquisition costs?
 acquisitionFees.capacitor.title=Does the person want to claim for any additional acquisition costs?
 acquisitionFees.personalRep.title=Are you including any additional acquisition costs?
 
+acquisitionFees.indirect.title=Do you want to claim any additional costs you paid when you got the shares?
+acquisitionFees.indirect.agent.title=Were there any additional acquisition costs?
+acquisitionFees.indirect.trust.title=Were there any additional acquisition costs?
+acquisitionFees.indirect.capacitor.title=Select yes if the person wants to claim any additional costs they paid when they got the shares
+acquisitionFees.indirect.personalRep.title=Were there any additional acquisition costs?
+
+
 acquisitionFees.helpText=For example, fees for surveyors or solicitors. If you shared these costs, you can only claim your share.
 acquisitionFees.agent.helpText=For example, fees for surveyors or solicitors. If your client shared these costs, they can only claim their share.
 acquisitionFees.trust.helpText=For example, fees for surveyors or solicitors. If the trust shared these costs, it can only claim its share.
 acquisitionFees.capacitor.helpText=For example, fees for surveyors or solicitors. If the person shared these costs, they can only claim their share.
 acquisitionFees.personalRep.helpText=For example, fees for surveyors or solicitors. If the person shared these costs, you can only include their share.
 
+acquisitionFees.indirect.helpText=For example, fees for surveyors or solicitors. If you shared these costs, you can only claim your share.
+acquisitionFees.agent.indirect.helpText=For example, fees for surveyors or solicitors. If your client shared these costs, they can only claim their share.
+acquisitionFees.trust.indirect.helpText=For example, fees for surveyors or solicitors. If the trust shared these costs, it can only claim its share.
+acquisitionFees.capacitor.indirect.helpText=For example, fees for surveyors or solicitors. If the person shared these costs, they can only claim their share.
+acquisitionFees.personalRep.indirect.helpText=For example, fees for surveyors or solicitors. If the person shared these costs, you can only include their share.
+
 acquisitionFees.rebased.title=Do you want to claim any additional costs you paid when you rebased the property?
 acquisitionFees.agent.rebased.title=Were there any additional rebasing costs?
 acquisitionFees.trust.rebased.title=Were there any additional rebasing costs?
 acquisitionFees.capacitor.rebased.title=Does the person want to claim for any additional rebasing costs?
 acquisitionFees.personalRep.rebased.title=Does the person want to claim for any additional rebasing costs?
 
+acquisitionFees.indirect.rebased.title=Do you want to claim any additional costs for working out the value of the shares on 5 April 2019?
```

Comment:
Just had a look through - I need to fix the hardcoded date

Reply:
does this still not need to be fixed?

Reply:
(and the ones below it)